pkg/api/kubegraph/edges.go

```diff
@@ -66,3 +73,88 @@ func AddAllExposedPodEdges(g osgraph.MutableUniqueGraph) {
 		}
 	}
 }
+
+func AddMountedSecretEdges(g osgraph.Graph, podSpec *kubegraph.PodSpecNode) {
+	//pod specs are always contained.  We'll get the toplevel container so that we can pull a namespace from it
+	containerNode := osgraph.GetTopLevelContainerNode(g, podSpec)
+	containerObj := g.GraphDescriber.Object(containerNode)
+
+	meta, err := kapi.ObjectMetaFor(containerObj.(runtime.Object))
+	if err != nil {
+		panic(err)
```

Comment:
Is a `panic` from a library call the best way to do error handling here? Why not return `error`?